apps/studio/src/features/settings/__tests__/utils.test.ts

```diff
@@ -0,0 +1,252 @@
+import { describe, expect, it } from "vitest"
+
+import {
+  BACKGROUND_COLOURS,
+  calculateRelativeContrast,
+  calculateRelativeLuminance,
+  generateColorPalette,
+  getPalette,
+  passesContrastCheck,
+  TEXT_COLOURS,
+  Theme,
+} from "../utils"
```

Comment:
nit:
- `Theme` imported as type
- `calculateRelativeContrast`, `calculateRelativeLuminance`, `generateColorPalette` imported but not used

Reply:
these got updated downstream but i'll fix it here too ps